src/actions/indexing/indexable-prepare-indexation-action.php

```diff
@@ -24,18 +26,28 @@ class Indexable_Prepare_Indexation_Action {
 	 */
 	private $date;
 
+	/**
+	 * The notification center.
+	 *
+	 * @var Yoast_Notification_Center
+	 */
+	private $notification_center;
```

Comment:
A decision was made earlier to make everything at least protected (or public if it is needed outside.)
```suggestion
	protected $notification_center;
```
```suggestion
	private $notification_center;
```